### PR TITLE
Remove hogan lib to remove unsafe eval error

### PR DESCRIPTION
**Pull request by bidoubiwa**
See https://github.com/algolia/docsearch/pull/773

### CODE REVIEW

**Comment by curquiza** [comment]
Is this PR breaking for the users?

**Comment by bidoubiwa** [comment]
It isn't! I tried all previous ways of doing things + all templates suggestions

**Review by alallema — APPROVED** [review]
LGTM! 🎉

File: package.json [hunk 1]
```diff
@@ -57,15 +57,14 @@
     "postcss-cli": "^7.1.2",
     "prettier": "^2.0.5",
     "pretty-bytes-cli": "^2.0.0",
-    "sass": "^1.43.4",
+    "sass": "^1.43.3",
```
Comment (alallema) [inline package.json:60]:
```suggestion
    "sass": "^1.43.4",
```

**Review by alallema — APPROVED** [review]

**Review by alallema — APPROVED** [review]
LGTM! 🍔

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**